nifi-api/src/main/java/org/apache/nifi/logging/LogMessage.java

```diff
@@ -26,18 +28,66 @@
 public class LogMessage {
 
     private final String message;
-    private final LogLevel level;
+    private final LogLevel logLevel;
     private final Throwable throwable;
-    private final long time;
+    private final FlowFile flowFile;
+    private final Object[] objects;
+    private long time;
 
     public static final String DATE_TIME_FORMAT = "yyyy-MM-dd HH:mm:ss.SSS";
     public static final String TO_STRING_FORMAT = "%1$s %2$s - %3$s";
 
-    public LogMessage(final long millisSinceEpoch, final LogLevel level, final String message, final Throwable throwable) {
-        this.level = level;
+    public static class Builder {
+
+        private long millisSinceEpoch;
+        private LogLevel level;
+        private String message;
+        private Throwable throwable;
+        private FlowFile flowFile;
+        private Object[] objects;
+
+        public Builder setMillisSinceEpoch(long millisSinceEpoch) {
```

Comment:
Similar to the comment in the other Builder, the pattern that we usually use in the NiFi codebase is to omit the `set` prefix on Builder methods and instead go with `.level(...).message(...).timestamp(...)` etc.